src/pool/pool_disjoint.c

```diff
@@ -523,8 +524,8 @@ static void disjoint_pool_print_stats(disjoint_pool_t *pool) {
         utils_mutex_unlock(&bucket->bucket_lock);
     }
 
-    LOG_DEBUG("current pool size: %zu",
-              disjoint_pool_get_limits(pool)->total_size);
+    LOG_DEBUG("current pool size: %llu",
+              (unsigned long long)disjoint_pool_get_limits(pool)->total_size);
```

Comment:
why?

Reply:
total_size is now uint 64bit as other atomics. This is needed for MacOS. Please suggest better change

Reply:
Great idea, but I don't want to mix PRI macros with what we currently have in our code. If we want to use PRI, it should be implemented in a whole project with a separate PR.

Reply:
ok, done